microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -20,16 +20,14 @@ ms.date: 11/22/2022
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.  -->
-# Get started analyzing runtime performance
+# Introduction to the Performance tool
```

Comment:
@mikehoffms, please do an edit pass on this doc and the second one that I'm working on. I'm flexible on the titles but before we had "Get started analyzing runtime performance" and "Analyze runtime performance" and I think we just need a clearer distinction. This one is really a tutorial/introduction/get started guide and I'd like the title to reflect that.

Reply:
The std todo list for changing a title:
* YAML field at top of .md
* h1 near top of .md
* toc entry
* toc index entry possibly add the old title, comma delim'd (often overlooked, check this)
* full text search repo for old title string (likely overlooked; todo)

Reply:
fixed 1x old "Get started analyzing..."